src/main/java/pl/javastart/task/models/Student.java

```diff
@@ -0,0 +1,38 @@
+package pl.javastart.task.models;
+
+public class Student extends Person {
+    private int index;
+    private Grade[] grades;
+
+    public Student(String firstName, String lastName, int index, int maxClassesAmount) {
+        super(firstName, lastName);
+        this.index = index;
+        this.grades = new Grade[maxClassesAmount];
+    }
+
+    public String getInfo() {
+        return String.format("%d %s", index, super.getInfo());
+    }
+
+    public String getGradesList() {
+        StringBuilder stringBuilder = new StringBuilder();
+        for (int i = 0; i < grades.length; i++) {
+            if (grades[i] != null) {
+                stringBuilder.append(grades[i].getInfo()).append("\n");
+            }
+        }
+        return stringBuilder.toString();
+    }
```

Comment:
Nazwa tej metody sugeruje, że w wyniku dostaniemy listę ocen, a w rzeczywistości dostajemy Stringa. Albo zamieniłbym ją tak, żeby zwracała tablicę Grade[], albo zmień nazwę na getGradesAsString
(przy okazji listy i tablice to dwie różne struktury danych, ale tego możesz jeszcze nie wiedzieć)